### DISCUSSION TITLE
Escaping quotes in attributes

### DISCUSSION THREAD

**Discussion by jacobc16** [markdoc/markdoc#151] [Q&A] [answered]
Is there a way to escape quotes in attributes, they are causing tags to not render.

The following below will cause an error to be thrown
```
{% tag txt="SomeText: "or something"" /%}
```

I have tried these with no success:
```
{% tag txt="SomeText: \"or something\"" /%}
```
```
{% tag txt='SomeText: "or something"' /%}
```
```
{% tag txt=`SomeText: "or something"` /%}
```

**Comment by jacobc16** [comment]
Ended up fixing the issue that this was a work around for. I did not understand how to change the p tag to a span but ended up doing this
```js
transform(node, config) {
	const attributes = node.transformAttributes(config);
	const children = node.transformChildren(config);

	for (let child of children) {
		child.name = 'span';
	}

	return new Tag(this.render, attributes, children);
}
```

**Comment by mfix-stripe** [comment, marked as answer]
@Walrusking16 escaping the the quote should work: 
![image](https://user-images.githubusercontent.com/62121649/182206953-4815e620-50ee-4134-a6c6-1d34aaf8bf89.png)
